_episodes/03-creating-containers.md

```diff
@@ -31,7 +31,7 @@ $ docker run hello-world
 ~~~
 Unable to find image 'hello-world:latest' locally
 latest: Pulling from library/hello-world
-1b930d010525: Pull complete 
+1b930d010525: Pull complete
```

Comment:
I'm completely happy to merge this sort of change, for convenience, but it just looks like whitespace. Is this possibly an editing artefact?

Reply:
```suggestion
1b930d010525: Pull complete
```

Reply:
I think this is an artifact from editing.  I can switch it back but it will make another commit.  However, when you merge this I think you should get the option to squash all of my commits into one so it may not matter.  Commit my suggestion above if that sounds good to you.